Sources/Commands/PackageCommands/ShowTraits.swift

```diff
@@ -0,0 +1,85 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2014-2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import Basics
+import CoreCommands
+import Foundation
+import PackageModel
+import PackageGraph
+import Workspace
+
+struct ShowTraits: AsyncSwiftCommand {
+    static let configuration = CommandConfiguration(
+        abstract: "List the available traits for a package.")
+
+    @OptionGroup(visibility: .hidden)
+    var globalOptions: GlobalOptions
+
+    @Option(help: "Show traits for the package id.")
```

Comment:
issues (possibly-blocking): by reading the help, this gives the impression any package ID can be sufficient, even if it's not a direct dependent on this current package.  Can we  update the help message to clarify which package ID's are supported by this argument?